src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_branch.yaml

```diff
@@ -0,0 +1,178 @@
+post:
+  summary: Branch a package.
+  description: |
+    Create a source link from a package of an existing project to a
+    new subproject of the requesters home project.
+    The default target is home:<user>:branches:<project>/<package>
```

Comment:
This line gets rendered in Swagger-UI as `The default target is home::branches:/`. Could we adjust it?